### PR TITLE
Added 404 error page

### PR DESCRIPTION
**Pull request by arpit3018**
1. Changed learn/templates/404.html file

### CODE REVIEW

File: learn/templates/404.html [hunk 1]
```diff
@@ -1,4 +1,619 @@
...
+            <h1>We couldn't find what you were looking for.</h1>
+
+            <p>Unfortunately the page you were looking for could not be found. It may be
+                temporarily unavailable, moved or no longer exist.</p>
+
+            <p>Check the URL you entered for any mistakes and try again. Alternatively, search
+                for whatever is missing or take a look around the rest of our site.</p>
+
+            <div id="navigation">
+                <a class="navigation" href="{% url 'index' %}">Home</a><a class="navigation" href="">Go Back</a>
```
Comment (mohitkh7) [inline learn/templates/404.html]: Go back URL is not working as expected.
It should go to previous valid URL, but it is reloading the same page again.

Comment (arpit3018) [reply learn/templates/404.html]: I have'nt added any href to go back url

**Comment by mohitkh7** [comment]
There should be a navbar on the top. Most of the websites 404 page have a navbar having atleast website name and logo.

File: learn/templates/404.html [hunk 2]
```diff
@@ -1,4 +1,619 @@
...
+            }
+        }
+
+        @media all and (max-width: 499px) {
+            #main-content h1 {
+                font-size: 28px;
+            }
+        }
+    </style>
+    <title>Start Learning</title>
```
Comment (mohitkh7) [inline learn/templates/404.html]: Page title should be something more relevant like `Easy Learning - 404 Page Not Found`

Comment (arpit3018) [reply learn/templates/404.html]: Okay

File: learn/templates/404.html (hunk 1 above)
Comment (arpit3018) [reply learn/templates/404.html]: Added Go back URL.

**Comment by arpit3018** [comment]
1. Added Go Back URL
2. Added Logo and Name at the left top corner of the page
3. Added hyperlink to the logo and the name 
4. Create a separate file for js and css and removed iniline css and js.

**Pull request merged**